grinder/interface.py

```diff
@@ -83,12 +83,24 @@ def parse_args(self) -> Namespace:
             default="queries.json",
             help="JSON File with Shodan queries",
         )
+        parser.add_argument(
+            "-target",
```

Comment:
Do we really need this flag? We already have `-oh`, we can put our target in it.

Reply:
good point